code/modules/research/designs/weapon_designs.dm

```diff
@@ -126,7 +126,7 @@
 	category = list("Weapons")
 
 /datum/design/smg
-	name = "Prototype Submachine Gun"
+	name = "NanoTrasen Saber SMG"
```

Comment:
MrPerson. This one is in the Protolathe. the \improper part isn't needed for it.